data_ingestion/common/gcf_household_income.py

```diff
@@ -0,0 +1,36 @@
+from .di_url_file_to_gcs import url_file_to_gcs
+
+
+def get_columns():
+  """Returns column names of SAIPE fields and their descriptions."""
+  return {
+    'COUNTY': 'County FIPS Code',
+    'GEOCAT': 'Summary Level',
+    'GEOID': 'State+County FIPS Code',
+    'SAEMHI_LB90': 'Median Household Income Lower Bound for 90% Confidence Interval',
+    'SAEMHI_MOE': 'Median Household Income Margin of Error',
+    'SAEMHI_PT': 'Median Household Income Estimate',
+    'SAEMHI_UB90': 'Median Household Income Upper Bound for 90% Confidence Interval',
+    'SAEPOVALL_LB90': 'All ages in Poverty, Count Lower Bound for 90% Confidence Interval',
+    'SAEPOVALL_MOE': 'All ages in Poverty, Count Margin of Error',
+    'SAEPOVALL_PT': 'All ages in Poverty, Count Estimate',
+    'SAEPOVALL_UB90': 'All ages in Poverty, Count Upper Bound for 90% Confidence Interval',
+    'SAEPOVRTALL_LB90': 'All ages in Poverty, Rate Lower Bound for 90% Confidence Interval',
+    'SAEPOVRTALL_MOE': 'All ages in Poverty, Rate Margin of Error',
+    'SAEPOVRTALL_PT': 'All ages in Poverty, Rate Estimate',
+    'SAEPOVRTALL_UB90': 'All ages in Poverty, Rate Upper Bound for 90% Confidence Interval',
+    'SAEPOVU_ALL': 'All Ages in Poverty Universe',
+    'STABREV': 'Two-letter State Postal abbreviation',
+    'STATE': 'FIPS State Code',
+    'YEAR': 'Estimate Year',
+  }
+
+
+def upload_household_income(url, gcs_bucket, filename):
+  """Uploads household income data from SAIPE to GCS bucket for all available years."""
+  for year in {1989, 1993, *range(1995, 2019)}:
```

Comment:
Just for readability, can we store the year range in a variable before iterating over them?